build.gradle

```diff
@@ -1,42 +1,29 @@
 buildscript {
-    dependencies {
-        // TODO(dsanduleac): remove once we apply baseline
-        classpath 'com.netflix.nebula:nebula-dependency-recommender:6.0.0'
+    repositories {
+        maven { url "http://palantir.bintray.com/releases" }
     }
 }
 
 plugins {
     id 'com.jfrog.bintray' version '1.8.4'
     id 'com.gradle.plugin-publish' version '0.9.9'
-    id 'com.palantir.configuration-resolver' version '0.2.0'
     id 'com.palantir.git-version' version '0.4.0'
     id 'com.palantir.idea-test-fix' version '0.1.0'
```

Comment:
this is a mystical plugin that we can't find the source for anymore - deletable?